docs/global/modules/samples/pages/use_case_2.adoc

```diff
@@ -2,18 +2,22 @@
 :navtitle: 2. Model server and web frontend
 
 This use case realizes the synchronization of model data between two web frontends and a model server.
+Optionally, an MPS project can be run and also synchronize its model data with the model server.
 
-One web frontend is the *Management application* implemented in https://vuejs.org/[Vue.js] and can modify the model content.
+One web frontend is the *Management application* implemented in Vue.js.
+It can be used to edit the model content.
 It uses the https://artifacts.itemis.cloud/service/rest/repository/browse/npm-open/%40modelix/model-client/[@modelix/model-client], the https://artifacts.itemis.cloud/service/rest/repository/browse/npm-open/%40modelix/vue-model-api/[@modelix/vue-model-api] and code generated by the  xref:core:reference/component-model-api-gen-gradle.adoc[Model API generator].
 The generated code provides type-safe access to model data and is derived from the xref:metamodel.adoc[MPS languages under `mps/languages`].
 
-The other web frontend is written in https://angular.io/[Angular] and only reads model data. It also uses the `model-client` and the generated type-safe TypeScript API.
+The other web frontend is the *Overview application* and is implemented in Angular.
```

Comment:
> we see "Management application" (MA) and "Overview application" (OA). MA is mentioned in the text, however OA is not mentioned. It would be great to mention OA in the Angular sentence.